.travis.yml

```diff
@@ -13,6 +13,7 @@ env:
 
 install:
   - npm install
+  - sudo pip install -r requirements.txt
```

Comment:
weird. it seems to work at https://github.com/GluuFederation/gluu-webui/blob/master/.travis.yml . Anyways use whatever works for now